covid19-dashboard/dc-covid19/src/DataCalculator.ts

```diff
@@ -1,90 +1,126 @@
 /**
  Copyright 2020 Google LLC
-
  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
-
  https://www.apache.org/licenses/LICENSE-2.0
-
  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
  */
 
-import {addOrSubtractNDaysToDate, getRealISODatesFromArrayOfDeltaDays} from './Utils'
+import {getRealISODatesFromArrayOfDeltaDays} from './Utils'
+import moment from "moment";
 
 type DataPerGeoIdPerDate = {string: {string: number}} | {}
 
 /**
  * Performs a calculationType iteratively on two dates.
  * @param data: input data in the form of dates->geoIds->value.
  * @param range: the range of dates to calculate the data for in ISO Format. Example: ["2020-01-01", "2020-01-10"]
- * @param deltaDays: Do we wanna compare the data every 2 days? every 7? Any number works.
  * @param calculationType: What type of calculation are we performing?
- * @param geoIdToPopulation: geoId->Population. After calculating the difference, we want to divide by total population
+ * @param deltaDays?: Do we wanna compare the data every 2 days? every 7? Any number works.
+ * @param geoIdToPopulation?: geoId->Population. After calculating the difference, we want to divide by total population
```

Comment:
I suggest not to do per-capita in this function. Instead have a separate function to do point-wise data conversion

Reply:
DataCalculator now only calculated the 'difference', 'increase', or absolute.
After DataCalculator finishes, then we calculate per-capita on the previous output.


**Should I separate absolutePerCapita and perCapita into in the config.json file?:**

```
'absolutePerCapita': ['absolute', 'perCapita'],
'perCapita: ['difference', 'perCapita']
```